logstreams/src/main/java/io/camunda/zeebe/logstreams/impl/log/LogStorageAppender.java

```diff
@@ -158,7 +116,8 @@ public String getName() {
 
   @Override
   protected void onActorStarting() {
-    actor.consume(writeBufferSubscription, this::onWriteBufferAvailable);
+    sequencer.registerConsumer(actor.onCondition("sequencer", this::tryWriteBatch));
+    actor.submit(this::tryWriteBatch);
```

Comment:
:upside_down_face: Could also just call `tryWriteBatch` directly on `onActorStarted`. The submitted call back is not called anyway until the actor is started, and we have the started hook for these kind of things.

Reply:
Yeah, but I always forget about this behavior and I'd imagine I might not be the only one so I'd prefer to keep it like this, just to be explicit.